plat/hisilicon/hikey960/hikey960_bl31_setup.c

```diff
@@ -141,6 +141,20 @@ static void hikey960_edma_init(void)
 	}
 }
 
+static void hikey960_iomcu_dma_init(void)
+{
+	int i;
+	uint32_t non_secure;
+
+	non_secure = IOMCU_DMAC_SEC_CTRL_INTR_SEC | IOMCU_DMAC_SEC_CTRL_GLOBAL_SEC;
+	mmio_write_32(IOMCU_DMAC_SEC_CTRL, non_secure);
+
+	/* channels 0-3 are reserved */
+	for (i = 4; i < IOMCU_DMAC_CHANNEL_NUMS; i++) {
+		mmio_write_32(IOMCU_DMAC_AXI_CONF(i), (1 << 6) | (1 << 18));
```

Comment:
Don't use magic numbers if you can. Add the definitions to the header file along with the others.

Reply:
These register are not publicly documented by Hisilicon, but I know them to be standard AXI bus access permissions. I will add the definitions.